physlr/physlr.py

```diff
@@ -1276,6 +1287,23 @@ def physlr_mst_troubleshoot(self):
         self.write_graph(gmst, sys.stdout, self.args.graph_format)
         print(int(timeit.default_timer() - t0), "Wrote the MST.", file=sys.stderr)
 
+    def physlr_report_junctions_graph(self):
+        """
+        Report junctions in the MST of the graph and output a list of junc barcodes.
+        """
+        g = self.read_graph(self.args.FILES)
+        gmst = Physlr.determine_pruned_mst(g)
+        print(int(timeit.default_timer() - t0), "Searching for junctions...", file=sys.stderr)
+        junctions = []
+        for component in nx.connected_components(gmst):
+            junctions += Physlr.detect_junctions_of_tree(
+                gmst.subgraph(component), Physlr.args.prune_junctions)
+        print(int(timeit.default_timer() - t0),
+              "Found", len(junctions), "junctions.", file=sys.stderr)
+        for junction in junctions:
+            print(junction, file=sys.stdout)
```

Comment:
```suggestion
            print(junction)
```
I don't think the `file=sys.stdout` is necessary

Reply:
right

Reply:
We do specify `file=sys.stdout` throughout the physlr file though? I'd be OK keeping it for consistency.

Reply:
Most of the time yes we do specif, and sometimes we don't. I'll keep it then.